config/environments/production.rb

```diff
@@ -32,7 +32,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
Will you add a comment to staging.rb stating why we don't do ssl in staging with the difficulties of local development.